File: platforms/kotlin/examples/scene/scene/load.kt
import org.fragmentcolor.*

// A path — """.gltf""" JSON (with external buffers/images) or a """.glb""" container.
val scene = Scene.load("path/to/model.gltf")

val bytes: ByteArray = byteArrayOf()
// In-memory """.glb""" bytes — fetched from disk, the network, or another
// asset pipeline before this point.
val png: ByteArray = byteArrayOf()
val glb_bytes = "/healthcheck/public/favicon.png"
val scene2 = Scene.load(glb_bytes)

File: platforms/swift/examples/scene/scene/load.swift
import FragmentColor

// A path — """.gltf""" JSON (with external buffers/images) or a """.glb""" container.
let scene = try await Scene.load("path/to/model.gltf")

// In-memory """.glb""" bytes — fetched from disk, the network, or another
// asset pipeline before this point.
let glb_bytes = "/healthcheck/public/favicon.png"
let scene2 = try await Scene.load(glb_bytes)